src/frontend/src/lib/i18n/en.json

```diff
@@ -29,13 +29,13 @@
 			"source_code_on_github": "Source code on GitHub",
 			"view_on_explorer": "View on explorer",
 			"source_code": "Source code",
-			"oisy_roadmap": "$oisy_name Roadmap",
+			"changelog": "Changelog",
 			"submit_ticket": "Submit a ticket"
 		},
 		"alt": {
 			"more_settings": "More settings",
 			"menu": "Your wallet address, settings, sign-out and external links",
-			"oisy_roadmap": "Open the $oisy_name roadmap",
+			"changelog": "Open the $oisy_name changelog",
```

Comment:
Nitpick: this alt can probably be more descriptive, e.g. "Open the changelog of $oisy_name on GitHub to review the latest updates"

Reply:
changed